server/src/main/java/com/lunchtime/service/impl/RestaurantServiceImpl.java

```diff
@@ -33,6 +40,14 @@ public Restaurant saveRestaurant(Restaurant restaurant) {
         return restaurantRepository.findByIsDeletedFalse(pageable);
     }
 
+    public Page<Restaurant> getRestaurantPageByUserId(Long userId, Pageable pageable) {
+        long userRoleId = personRepository.findByRoleId(userId);
+        if (userRoleId == OWNER_ROLE_ID) {
+            return restaurantRepository.findByOwnerRestList(userId, pageable);
+        }
+        return null;
+    }
```

Comment:
I'm not sure if we should store `OWNER_ROLE_ID` here. We can get this value from frontend constants right now. But, honestly, I'd get thid id value from Java Object Enums that can be stored in DB with its persisting id. So, we can check users role withoud access to DB in line 44. Haven't implemented smth like that, but I think this is possible.

Reply:
maybe we could get it from frontend, but in case for example in postman I can put some other number, and it wouldn'd work
Earlier we already had CategoryEnum for event, and we moved this to DB. This is a same situation)